third_party/libgcc/udivmoddi4.c

```diff
@@ -23,11 +23,15 @@ a copy of the GCC Runtime Library Exception along with this program;
 see the files COPYING3 and COPYING.RUNTIME respectively.  If not, see
 <http://www.gnu.org/licenses/>.  */
 
-/* This is extracted from gcc's libgcc/libgcc2.c with these typedefs added: */
-typedef short Wtype;
-typedef int DWtype;
-typedef unsigned int UWtype;
-typedef unsigned long long UDWtype;
+#include <stdint.h>
+
+/* This is extracted from gcc's libgcc/libgcc2.c with these typedefs added.
+   Note that for the targets we care about LIBGCC2_UNITS_PER_WORD == 4.
```

Comment:
Would be nicer for clarity to mention the targets, so that we know if/when this needs to be updated.

Reply:
Done